frontend/webadmin/modules/uicommonweb/src/main/java/org/ovirt/engine/ui/uicommonweb/models/vms/VmModelBehaviorBase.java

```diff
@@ -1626,18 +1626,23 @@ protected void updateGraphics(Guid id) {
     }
 
     /**
-     * allows to enable numa models in all derived behaviors
-     * use updateNumaEnabledHelper in each behavior that requires numa
+     * allows to update numa models in all derived behaviors
```

Comment:
it's been there before - but numa models?

Reply:
yes, everything is a model in the UI. I do not think that comment is very important though. We could remove it completely if you wish.

Reply:
ah it refers to the UI models.. ok